tests/NHSD.BuyingCatalogue.Solutions.API.UnitTests/IntegrationsControllerTests.cs

```diff
@@ -0,0 +1,52 @@
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NHSD.BuyingCatalogue.Solutions.API.Controllers;
+using NHSD.BuyingCatalogue.Solutions.Contracts;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Queries;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Solutions.API.UnitTests
+{
+    [TestFixture]
+    public sealed class IntegrationsControllerTests
+    {
+        private Mock<IMediator> _mockMediator;
+
+        private IntegrationsController _controller;
+
+        private const string SolutionId = "Sln1";
+
+        [SetUp]
+        public void Setup()
+        {
+            _mockMediator = new Mock<IMediator>();
+            _controller = new IntegrationsController(_mockMediator.Object);
+        }
+
+        [Test]
+        public async Task ShouldGetIntegrations()
```

Comment:
Use a testcase instead. Do one where it is null and one where it is a string

Reply:
also added for roadmap